packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts

```diff
@@ -90,10 +91,10 @@ export class BasicTracerProvider implements TracerProvider {
     }
   }
 
-  getTracer(name: string, version?: string): Tracer {
+  getTracer(name: string, version?: string, options?: TracerOptions): Tracer {
     const key = `${name}@${version || ''}`;
     if (!this._tracers.has(key)) {
-      this._tracers.set(key, new Tracer({ name, version }, this._config, this));
+      this._tracers.set(key, new Tracer({ name, version, schemaUrl: options?.schemaUrl }, this._config, this));
```

Comment:
shouldn't we add the `schemaUrl` into the key?
Otherwise it doesn't make much sense to pass a schemaUrl as you may get an already created `Tracer` with a different one.

Reply:
Done 30cc71fa9b8f9e0de2b2373f32b3ffdf8a4ed74f